src/Main.js

```diff
@@ -9,7 +9,7 @@ import Stuff from "./Stuff";
 import Contact from "./Contact";
 import Login from "./Login/Login";
 import Navbar from './Navbar'
-import Footer from './Footer'
+import Footer from './footer/Footer'
```

Comment:
I suggest you move footer to common components if you use it in more than 1 page or move to specific page module (main or home module)